plugins/importexport/crossref/CrossRefExportPlugin.inc.php

```diff
@@ -120,19 +120,22 @@ function displayIssueList(&$templateMgr, &$journal) {
 		$this->registerDaoHook('IssueDAO');
 		$issueIterator =& $issueDao->getPublishedIssues($journal->getId(), Handler::getRangeInfo('issues'));
```

Comment:
When reusing variable names, generally it's a good idea to `unset($issueIterator);` to avoid reference wierdness.